Source/DataSources/ModelGraphics.js

```diff
@@ -40,7 +44,7 @@ define([
      * @param {Object} [options] Object with the following properties:
      * @param {Property} [options.uri] A string Property specifying the URI of the glTF asset.
      * @param {Property} [options.show=true] A boolean Property specifying the visibility of the model.
-     * @param {Property} [options.scale=1.0] A numeric Property specifying a uniform linear scale.
+     * @param {Property} [options.scale=new Cartesian(1.0, 1.0, 1.0)] A Property specifying a scale value represented by a {@link Cartesian3}.
```

Comment:
The current default is good, I think the description should mention that a number can also be provided rather than just a Cartesian3.